src/components/MessageReviewPanel/ReviewTooltip.tsx

```diff
@@ -0,0 +1,80 @@
+import { Tooltip } from "antd";
+import React from "react";
+import { MessageStatus } from "src/api/graphql";
+import { formatDate } from "src/utils";
+import { TMessageItem } from "./MessageCard";
+
+interface Props {
+  message: TMessageItem;
+  children?: React.ReactNode | React.ReactNode[]
+}
+
+const ReviewTooltip = ({ message, children }: Props) => {
+  const MESSAGE_AUTO_DELAY_MINUTES = 30;
+  const reviewer = message.reviewedBy ? `${message.reviewedBy?.firstName} ${message.reviewedBy?.lastName}` : "Unknown";  // This only occurs from a data error.
```

Comment:
the ternary should be enough:

```suggestion
  const reviewer = message.reviewedBy ? `${message.reviewedBy.firstName} ${message.reviewedBy.lastName}` : "Unknown";  // This only occurs from a data error.
```